tasks/monitoring.yml

```diff
@@ -0,0 +1,26 @@
+- name: Add host in Zabbix UI
+  block:
+  - name: Configuring agent in Zabbix server
+    delegate_to: "{{ zabbix_api_srv }}"
+    when: zabbix_api_srv is defined
+    ansible.builtin.include_role:
+      name: zabbix-server
+      tasks_from: agent_config
+      vars:
+        zabbix_templates: "{{ fmn_zabbix_templates }}"
+        zabbix_groups: "{{ fmn_zabbix_groups }}"
+    tags:
+    - monitoring
+
+- name: Zabbix check for FMN cert expiry
+  ansible.builtin.template:
+    src: zabbix-check-fmg-cert.sh.j2
+    dest: /usr/lib/zabbix/zabbix-check-fmn-cert.sh
+    mode: 0750
+
+- name: Zabbix cron jobs
+  ansible.builtin.cron:
+    name: FMN Cert monitoring
+    job: /usr/lib/zabbix/zabbix-check-fmn-cert.sh
+    minute: "0"
+    hour: "0"
```

Comment:
I'd probably just use `hour: '*'` to have hourly stats (also to not have to wait one day to have alert disappearing from zabbix (and just removing hour would even be equivalent to `hour: '*'` )

Reply:
Done